### PR TITLE
Add neutis flasher wor Windows

### PR DESCRIPTION
**Pull request by smirnov1993**

### CODE REVIEW

File: docs/flasher/gui.md [hunk 1]
```diff
@@ -21,15 +18,43 @@ Flasher has user friendly graphical interface.
...
+#### Windows
+
+* Download [Neutis Flasher for Windows](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool-v1.1.1-setup.exe)
+* Launch downloaded executable and follow the installation wizard
+* You will be asked to install `Microsoft Visual C++ Redistributable`. Accept the licence agreement and install it
+* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop
+
+!!! note
+    During the first launch of the app Windows may ask you for network access, bacause Flashers' inner architecture is based on client-server structure.
+    The application doesn't sent any data over the internet.
```
Comment (staroselskii) [inline docs/flasher/gui.md]: over the Internet

File: docs/flasher/gui.md [hunk 2]
```diff
@@ -21,15 +18,43 @@ Flasher has user friendly graphical interface.
...
 
+#### Windows
+
+* Download [Neutis Flasher for Windows](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool-v1.1.1-setup.exe)
+* Launch downloaded executable and follow the installation wizard
+* You will be asked to install `Microsoft Visual C++ Redistributable`. Accept the licence agreement and install it
+* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop
+
+!!! note
+    During the first launch of the app Windows may ask you for network access, bacause Flashers' inner architecture is based on client-server structure.
```
Comment (staroselskii) [inline docs/flasher/gui.md]: Flasher's

File: docs/flasher/gui.md (hunk 1 above)
Comment (smirnov1993) [reply docs/flasher/gui.md]: fixed

File: docs/flasher/gui.md [hunk 3]
```diff
@@ -21,15 +18,43 @@ Flasher has user friendly graphical interface.
...
+#### Windows
+
+* Download [Neutis Flasher for Windows](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool-v1.1.1-setup.exe)
+* Launch downloaded executable and follow the installation wizard
+* You will be asked to install `Microsoft Visual C++ Redistributable`. Accept the licence agreement and install it
+* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop
+
+!!! note
+    During the first launch of the app Windows may ask you for network access, bacause Flasher's inner architecture is based on client-server structure.
+    The application doesn't sent any data over the Internet.
```
Comment (AD-Aleksandrov) [inline docs/flasher/gui.md]: send - I guess

Comment (smirnov1993) [reply docs/flasher/gui.md]: fixed

**Pull request merged**